hardhat.config.ts

```diff
@@ -69,6 +95,15 @@ extendConfig((config: HardhatConfig) => {
         ],
       },
     };
+    if (process.env.HARDHAT_FORK_NETWORK) {
+      config.external.deployments!.hardhat = [
```

Comment:
Would this alternative work?

`config.external.deployments!.hardhat = config.external.deployments[process.env.HARDHAT_FORK_NETWORK]`

Reply:
If a network is not being forked, we don't want to use external deployments. For example testing